docs/current_docs/api/snippets/functions/arguments-float/go/main.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
It looks like the code snippets are duplicated between the args and retvals examples, for each language. Instead of this, I suggest changing the snippet directory names to reflect dual use e.g. `arguments-return-values-float` and then reusing/importing the same snippet for both the args and retvals examples.

Reply:
Done in: [f538507](https://github.com/dagger/dagger/pull/9408/commits/f538507862a018c8a3ae541e03ed41030d235ab2)